### DISCUSSION TITLE
[Help]: Is it possible to create nested carousel with dynamic/virtual slides? #501

### DISCUSSION THREAD

**Discussion by or2008** [davidjerleke/embla-carousel#502] [Help]
in react version, is it possible to create a nested carousel that utilizes both the X and Y axes and accepts dynamic slides based on the scroll direction.

for example if I swipe right, fetch and render slide 1, if I swipe down, fetch and render another slide etc..

It's important to note that pre-rendering all the slides isn't feasible in this case, as the slides are determined by the user's path, similar to a linked list or binary tree structure.

appreciate any help

**Comment by davidjerleke** [comment]
Thank you for your question @or2008. Embla doesn't support both `X` and `Y` scrolling at the same time. This is not planned as a new feature.
